20210325/MINUTES.md

```diff
@@ -0,0 +1,133 @@
+        - [ ] Users can break Sardana by messing with configuration in Tango DB, diagnostic script should be created to check that.
+            - MAX IV created a script (called Sherlock) to check Pool configuration - it check if all dependent elements exist (by ID)
+            - still in development - showing missing elements aliases instead of just IDs might be a nice feature
+        - [ ] Configure CI on buster (problems with test suite on Python 3.6): status update
+            - No update
+        - [ ] Verify if Sardana axis/controller attribute are memorized by default
+            - No update
+        - [ ] Hang scan due to the counter/timer controller timeout in StateOne() - How to deal with exceptions in the controllers: Review the docs?
+            - No update
+
+6. Overview of current developments / PR distribution
+    - Discussion: how to handle this section not to go through the list of recurrent issues/PRs?
+        - do not make a static list in the agenda
+        - add here issues only that need discussion
+        - one can add points in this section during the meeting (if comment/help/review is necessary)
+    - pre- and post-move hook [#1480](https://github.com/sardana-org/sardana/pull/1480)
+        - some issues found - Daniel, Zibi and Teresa will stay in touch to solve them
+    - Creating pseudocounters from pseudocounters 
+        - it works until one restarts Sardana - then because of the order of adding the elements during startup, Sardana is missing them and 
+        can't start at all
+        - problem seem to occur only on real elements - on dummy ones it works
+    - Apply position formatting to the limits [#1530](https://github.com/sardana-org/sardana/pull/1530)
+        - docstring style is messing the sphinx build: change style on enable napoleon extension (if accepted) [#1533](https://github.com/sardana-org/sardana/pull/1533)  
+    - Docstring styles [#1535](https://github.com/sardana-org/sardana/issues/1535)
+        - docstring should be consistent
+        - we keep the discussion open to see if there is any interest in the community to use a different docstring style than the reStructuredText (reST)
+    - Move regscans to scans [#1540](https://github.com/sardana-org/sardana/pull/1540)
+        - do we need backporting?
+            - in DESY they use custom names
+    - Set stop flags for each element in multiple elements movement [#1539](https://github.com/sardana-org/sardana/pull/1539)
+        - at the moment bug fix is merged to develop after manual tests
+        - automatic tests based on PyTest need some more attention and will be merged later not to delay the Release
+    - Execute post-scan also in case of stop, abort and exception [#1538](https://github.com/sardana-org/sardana/pull/1538)
+        - post-scan hooks are performed in case of an exception - it is good
+        - at the moment they are not called in a case of stopping and aborting
```

Comment:
I don´t knoe if we should add some more explanation here:
 they are not called in case of stop or abort since no macros can run after a ctrl-c.
 Possible solutions would be allow only the post-scan macro to run after the ctrl-c or
use a callable instead. The preferred solution is the first one, but it has to be evaluated.
The second one is already used at DESY to overcome that problem.

Reply:
Thanks for the suggestion! I've changed that point according to it.